src/main/java/io/fabric8/maven/docker/util/EnvUtil.java

```diff
@@ -308,7 +308,7 @@ public static String findRegistry(String ... checkFirst) {
     }
 
     public static File prepareAbsoluteOutputDirPath(MojoParameters params, String dir, String path) {
-        return prepareAbsolutePath(params, new File(params.getOutputDirectory(), dir).toString(), path);
+        return prepareAbsolutePath(params, new File(new File(params.getProject().getBuild().getDirectory(), params.getOutputDirectory()), dir).toString(), path);
```

Comment:
tbh, I just would leave the call to `prepareAbsolutePath `here as it was and recommend to use `${build.directory}/docker` in the pom.xml as configuration for the outputDir.